cli/tests/test_help_command.rs

```diff
@@ -84,3 +79,145 @@ fn test_help() {
+
+    Usage: jj [OPTIONS] <COMMAND>
+
+    Commands:
+      abandon           Abandon a revision
+      backout           Apply the reverse of a revision on top of another revision
+      bookmark          Manage bookmarks
+      commit            Update the description and create a new change on top [aliases: ci]
+      config            Manage config options
+      describe          Update the change description or other metadata [aliases: desc]
+      diff              Compare file contents between two revisions
+      diffedit          Touch up the content changes in a revision with a diff editor
+      duplicate         Create a new change with the same content as an existing one
+      edit              Sets the specified revision as the working-copy revision
+      evolog            Show how a change has evolved over time [aliases: evolution-log]
+      file              File operations
+      fix               Update files with formatting fixes or other changes
+      git               Commands for working with Git remotes and the underlying Git repo
+      help              Print this message or the help of the given subcommand(s)
+      init              Create a new repo in the given directory
+      interdiff         Compare the changes of two commits
+      log               Show revision history
+      new               Create a new, empty change and (by default) edit it in the working copy
+      next              Move the working-copy commit to the child revision
+      operation         Commands for working with the operation log [aliases: op]
+      parallelize       Parallelize revisions by making them siblings
+      prev              Change the working copy revision relative to the parent revision
+      rebase            Move revisions to different parent(s)
+      resolve           Resolve a conflicted file with an external merge tool
+      restore           Restore paths from another revision
+      root              Show the current workspace root directory
+      show              Show commit description and changes in a revision
+      simplify-parents  Simplify parent edges for the specified revision(s)
+      sparse            Manage which paths from the working-copy commit are present in the working copy
+      split             Split a revision in two
+      squash            Move changes from a revision into another revision
+      status            Show high-level repo status [aliases: st]
+      tag               Manage tags
+      util              Infrequently used commands such as for generating shell completions
+      undo              Undo an operation (shortcut for `jj op undo`)
+      version           Display version information
+      workspace         Commands for working with workspaces
+        
+
+    Help Categories:
+      revsets           Documentation about revsets
+      tutorial          Show a tutorial to get started with jj
```

Comment:
I would move "Help Categories" to bottom because it's not related to the command.

I found the default template in clap source:
https://github.com/clap-rs/clap/blob/master/clap_builder/src/output/help_template.rs#L64

If we don't have to insert "Help Categories" here, maybe we can just use it?

Reply:
or `.after_long_help()`?
https://docs.rs/clap/latest/clap/struct.Command.html#method.after_long_help

Reply:
Oh, much better

Reply:
One small thing. With `after_long_help` we can't use `{tab}`.